utils/config/daos_server.yml

```diff
@@ -336,6 +345,15 @@
 #  # - "kdev" to use a kernel block device, bdev_{size,number} ignored
 #  # Immutable after reformat.
 #
+#  # If Volume Management Devices (VMD) are used, then the enable_vmd flag
+#  # needs to be set. The bdev_class will remain the default "nvme" type,
+#  # and similar to NVMe, bdev_list will include the NVMe SSD address behind
+#  # the VMD. vmd_include also needs to be set to the VMD address in order for
+#  # the BDF address of the NVMe VMD SSD (ie "5d0505:01:00.0") to be parsed.
+#  bdev_class: nvme
+#  enable_vmd: true
```

Comment:
probably this option is redundant when we already have 'vmd_include' option above?

Reply:
It just made it easier when parsing the config since the vmd_include is a server param, and enable_vmd is a bdev/storage param. @tanabarr @mjmac I'm open to different ways to make this easier to use in the server config, just took my best stab at it.